### ISSUE TITLE
change training iteration output to progress bar style

### ISSUE THREAD

**Issue by ihorizons2022** [MrNeRF/gaussian-splatting-cuda#24]
it is more easier to observer the iterations per second.

**Comment by MrNeRF** [comment]
Agree. We should change this and what is reported. Perhaps you like to provide a reference implementation?

**Comment by ihorizons2022** [comment]
i think implementation in https://github.com/graphdeco-inria/gaussian-splatting is fine.
or tqdm in python

**Comment by MrNeRF** [comment]
Sure. But unfortunately, this won't be high priority for now at least not for me. That's why I am asking. But I would be very happy if you or someone comes up with an implementation in C++. Would be a really nice addition. I mark this issue accordingly. Maybe someone will grab it.

Thank you for bringing this up!

**Comment by ihorizons2022** [comment]
No problem

**Comment by MrNeRF** [comment]
Here you go: https://github.com/MrNeRF/gaussian-splatting-cuda/pull/26
Thx @paulmelis

**Comment by ihorizons2022** [comment]
Nice, I will have a try immediately.
Next expectation is a more convenient viewer or editor.

Cheers
